src/src/main/java/controller/AccessServlet.java

```diff
@@ -0,0 +1,18 @@
+package controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(name = "AccessServlet", value = "/access")
+public class AccessServlet extends HttpServlet {
+
+    protected void doGet(HttpServletRequest request,
+                         HttpServletResponse response) throws ServletException, IOException {
+        request.setAttribute("accessDenied", "Not enough rights");
+        request.getRequestDispatcher("/users").forward(request, response);
```

Comment:
if user hasn't enough rights, I think that we shouldn't redirect it to /users page